HTML/adithyanandakumar.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+    
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Counter App</title>
+</head>
+<body>
+    <div class="container">
+        <div class="lessButton">
+            <button class="less"><i class="fas fa-arrow-down fa-8x"></i></button>
+        </div>
+        <div class="numbers">
+            <div class="mainNum">10</div>
+        </div>
+
+        <div class="moreButton">
+            <button class="more"><i class="fas fa-arrow-up fa-8x"></i></button>
+        </div>
+    </div>
+
+    <script src="JAVASCRIPT/adithyanandakumar.js"></script>
```

Comment:
Adithya nice to see you working on this repo, but theirs a minor change requested from your side. 
In the src of the script, you first need to come out of the HTML folder and you need to enter in JAVASCRIPT folder, the correct file destination for the same would be ```../JAVASCRIPT/adithyanandakumar.js```, the ".." are for coming out of the folder you are working in.
Please make the changes so that I can merge them to my repo.